packages/webdriverjs/tests/example/webdriverjs-example.ts

```diff
@@ -1,12 +1,23 @@
 import { assert } from 'chai';
 import { Builder, WebDriver } from 'selenium-webdriver';
+import AxeBuilder from '../../src';
+import testListen = require('test-listen');
+import { Server, createServer } from 'http';
 import * as chrome from 'selenium-webdriver/chrome';
-import AxeBuilder from '@axe-core/webdriverjs';
+import * as express from 'express';
+import * as path from 'path';
 
-describe('Deque Webdriverjs example', () => {
+describe('@axe-core/webdriverjs example', () => {
   let driver: WebDriver;
+  let server: Server;
+  let addr: string;
+
+  beforeEach(async () => {
+    const app = express();
+    app.use(express.static(path.resolve(__dirname, '..', 'fixtures')));
+    server = createServer(app);
+    addr = await testListen(server);
```

Comment:
Doesn't seem like this should run each test. Better to just use a `before` here.